libdnf/rpm/solv_sack.cpp

```diff
@@ -427,10 +462,26 @@ void SolvSack::Impl::load_repo(Repo & repo, bool build_cache, LoadRepoFlags flag
     considered_uptodate = false;
 }
 
+
 void SolvSack::load_repo(Repo & repo, bool build_cache, LoadRepoFlags flags) {
```

Comment:
Please could you add TODO? I suggest that we can replace -- bool build_cache, LoadRepoFlags flags -- by configuration option. The function like it is is 100% fine for API users, but cannot be modified from CLI.